components/MermaidDiagram.module.css

```
@@ -147,3 +147,45 @@
+.mermaidTarget :global(.box text),
+.mermaidTarget :global(.box tspan),
+.mermaidTarget :global(.done .nodeLabel),
+.mermaidTarget :global(.done .nodeLabel *),
+.mermaidTarget :global(.done text),
+.mermaidTarget :global(.done tspan) {
+    color: #000000 !important;
+    fill: #000000 !important;
+}
+
+/* ── Accelerate カラーコードフォールバック (classDef を持たない既存図の互換用) ── */
+.mermaidTarget :global(.node[style*="FAF1DF" i] .nodeLabel),
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)